src/Dscp/CLI/Common.hs

```diff
@@ -43,3 +53,27 @@ versionOption :: Parser (a -> a)
 versionOption = infoOption ("disciplina-" <> (showVersion version)) $
     long "version" <>
     help "Show version."
+
+----------------------------------------------------------------------------
+-- ZMQ TCP
+----------------------------------------------------------------------------
+
+-- | Parse peers to connect to.
+peersParser :: Parser (Set ZTNodeId)
+peersParser =
+    fmap Set.fromList $
+    many $
+    option (eitherReader parseZTNodeId)
+           (long "peer" <> metavar "HOST:PORT1:PORT2" <> help "Peer(s) we should connect to")
```

Comment:
Does ZMQ require that exactly two ports are provided? If not (if any number of ports equal to one or more is allowed), then this `metavar` should be like `HOST:PORT1[:PORTx]*`

Reply:
Exactly two. We might discuss this later, but I obviously anticipated the confusion here. This model is much better and reliable from the ZMQ standpoint -- one socket is ROUTER (for p2p communication), other is PUB (subscriptions).

Reply:
I've tested timeouts, they work as expected.